scene/gui/text_edit.h

```diff
@@ -822,6 +822,7 @@ class TextEdit : public Control {
 	int get_last_full_visible_line_wrap_index() const;
 
 	int get_visible_line_count() const;
+	int get_visible_line_count_from(int p_from, int p_to) const;
```

Comment:
The name is a bit awkward but I don't know if I have any better suggestion.

Maybe `get_visible_line_count_in_range(from, to)`? Or just `get_visible_line_count_range(from, to)`?

Reply:
Yeah struggled with a good name here went with `_from` to align with `get_next_visible_line_offset_from`, but  `get_visible_line_count_in_range` sound better to me.